src/components/UI/LanguageSelector/index.tsx

```diff
@@ -99,7 +99,7 @@ export default function LanguageSelector({
     <select
       data-testid="languageSelector"
       onChange={(event) => handleLanguageChange(event.target.value)}
-      className={className}
+      className={className + ' is-drop-down'}
```

Comment:
```suggestion
      className={`${className} is-drop-down`}
```

Reply:
bruh package to join classes is that like isOdd and isEven

Reply:
> 

done